frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/base/built_in_page_widget.dart

```diff
@@ -54,10 +54,14 @@ class _BuiltInPageWidgetState extends State<BuiltInPageWidget> {
           if (board != null) {
             return _build(context, board);
           }
+          return const SizedBox.shrink();
+        } else if (snapshot.hasError) {
+          return const SizedBox.shrink();
+        } else {
+          return const Center(
+            child: CircularProgressIndicator(),
+          );
         }
```

Comment:
```suggestion
        }
        return const Center(
          child: CircularProgressIndicator(),
        );
```
The ending else statement is redundant, it's up to personal preference though.